guides/plugins/plugins/framework/message-queue/add-message-handler.md

```diff
@@ -173,6 +173,24 @@ framework:
 
 You can route messages by their classname and use the asterisk as a fallback for all other messages. If you specify a list of transports the messages will be routed to all of them. For more information on this check the [Symfony docs](https://symfony.com/doc/current/messenger.html#routing-messages-to-a-transport).
 
+#### Routing overwrites
+
+By default, all messages that implement the `AsyncMessageInterface` will be routed to the `async` transport. THe default symfony config detailed above will only let you add additional routing to those messages, however if you need to overwrite the additional routing you can do so by adding the following to your `shopware.yaml`:
+
+```yaml
+shopware:
+  messenger:
+    routing_overwrite:
+      Shopware\Core\Framework\DataAbstractionLayer\Indexing\EntityIndexingMessage: entity_indexing
```

Comment:
double backslash or not? idk because of escaping

Reply:
used single ticks as in the docs before